graphql-dgs-pagination/src/main/kotlin/com/netflix/graphql/dgs/pagination/DgsPaginationTypeDefinitionRegistry.kt

```diff
@@ -28,8 +29,13 @@ class DgsPaginationTypeDefinitionRegistry {
     fun registry(schemaRegistry: TypeDefinitionRegistry): TypeDefinitionRegistry {
         val definitions = schemaRegistry.types()
         val connectionTypes = parseConnectionDirective(definitions.values.toMutableList())
+
         val typeDefinitionRegistry = TypeDefinitionRegistry()
         typeDefinitionRegistry.addAll(connectionTypes)
+        val directive = DirectiveDefinition.newDirectiveDefinition().name("connection")
+            .directiveLocation(DirectiveLocation.newDirectiveLocation().name(Introspection.DirectiveLocation.OBJECT.name).build()).build()
```

Comment:
Should this first check if it was already added manually to the schema?

Reply:
Added the check.